src/client/app/components/LineChartComponent.tsx

```diff
@@ -86,8 +95,17 @@ export default function LineChartComponent() {
 				}}
 				config={{
 					responsive: true,
-					displayModeBar: false,
-					// Current Locale
+					displayModeBar: true,
+					modeBarButtonsToRemove: listOfButtons,
+					modeBarButtonsToAdd: [{
+						name: 'toggle-options',
+						title: 'Toggle Options',
```

Comment:
This is not internationalized. Look at useTranslate (src/client/app/components/unit/CreateUnitModalComponent.tsx has an example) for how to do the code. You also need to add the used key and text to data.ts in all three languages. When it works the text will change when the language is changed. This applies across all the pages with this icon. Let me know if you need help on this.